src/semgrep_agent/main.py

```diff
@@ -90,6 +90,12 @@ def url(string: str) -> str:
     type=int,
     help="Your semgrep.dev deployment ID (requires --publish-token)",
 )
+@click.option(
+    "--disable-metrics",
```

Comment:
The correct (TM) way to do this is

```suggestion
    "--enable-metrics/disable-metrics",
    envvar="ENABLE_METRICS",
    default=False
```

see https://click.palletsprojects.com/en/8.0.x/options/#boolean-flags